packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts

```diff
@@ -65,6 +66,13 @@ export class BrowserKeyboardLayoutProvider implements KeyboardLayoutProvider, Ke
         }
     }
 
+    private isAbleToGetLayoutMap(): boolean {
+        if (browser.isChrome && window.top !== window && window.location.protocol === 'https:') {
```

Comment:
Does this problem apply only to Chrome? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Keyboard/getLayoutMap), the Opera browser also supports this API.

Should we wrap the code with try-catch to be sure we get all corner cases and supporting browsers?